DemoCommon/Utils/Database/Operations/CreateSampleDataOperation.cs

```diff
@@ -13,7 +14,7 @@ public RavenCommand GetCommand(DocumentConventions conventions, JsonOperationCon
             return new CreateSampleDataCommand();
         }
 
-        private class CreateSampleDataCommand : RavenCommand
+        private class CreateSampleDataCommand : RavenCommand, IRaftCommand
```

Comment:
why is this needed?

Reply:
Each cluster related command needs to provide unique ID. We have introduced it in https://issues.hibernatingrhinos.com/issue/RavenDB-13556 and all our commands in client API implements it. Also `CreateSampleDataCommand ` in Tests:

https://github.com/ravendb/ravendb/blob/8fa6b32cc74a0033604972e8d789ce1bb2a96c16/test/Tests.Infrastructure/CreateSampleDataOperation.cs#L17

Reply:
Oh, because we are setting the revisions in the handler, okay.